draft-ietf-quic-recovery.md

```diff
@@ -79,13 +79,40 @@ recovery, and where applicable, attributes the TCP equivalent in RFCs,
 Internet-drafts, academic papers, and/or TCP implementations.
 
 
-## Notational Conventions
+# Conventions and Definitions
 
 The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
 "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this
 document are to be interpreted as described in BCP 14 {{!RFC2119}} {{!RFC8174}}
 when, and only when, they appear in all capitals, as shown here.
 
+Definitions of terms that are used in this document:
+
+Sender:
+
+: The endpoint sending QUIC packets.
+
+Receiver:
+
+: The endpoint receiving QUIC packets and sending acknowledgements in
+  response.
+
+ACK-only:
+
+: Any packet containing only an ACK or ACK_ECN frame. The rest of this
+  document uses "ACK frames" to refer to both ACK and ACK_ECN frames.
+
+In-flight:
+
+: Packets are considered in-flight because they have been sent
+  and neither acknowledged or considered lost, and they are not
+  ACK-only.
+
+Retransmittable:
+
+: Packets that contain frames besides ACK, ACK_ECN, or PADDING elicit
+  an ACK from the receiver and are called retransmittable.
```

Comment:
I think you should define retransmittable frames, and packets containing a retransmittable frame as a retransmittable packet.